openlibrary/templates/books/edit/edition.html

```diff
@@ -208,8 +208,9 @@
                                     $for role in edition_config.roles:
                                         <option>$role</option>
 
-                                    <!-- <option>---</option> -->
-                                    <!-- <option value="__add__">$_("Add a new role")</option> -->
+                                    $if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
+                                        <option>--- $_("Librarian Only")</option>
```

Comment:
```suggestion
                                        <option disabled>--- $_("Librarian Only")</option>
```
Prevents "--- Librarian Only" from being selected accidentally (probably not much of a concern).